docs/decisions/0005-optional-import-of-confluent-kafka.rst

```diff
@@ -0,0 +1,43 @@
+4. Optional import of confluent-kafka
+====================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+* `confluent-kafka`_ is a library written and maintained by Confluent, our managed instance provider (see :doc:`0004-kafka-managed-hosting`). The library abstracts out the work for sending and receiving events to and from the Kafka cluster and converting them into message objects.
+* confluent-kafka in turn is a wrapper around a C library called `librdkafka`_ (distributed as `librdkafka_dev`)
+* librdkafka-dev does not currently have a compiled binary available for Linux/aarch64
+
+As a result of the points above, if a package includes a dependency on confluent-kafka, developers will not be able to install the package in Linux/aarch64 environments.
+
+.. _confluent-kafka: https://github.com/confluentinc/confluent-kafka-python
+.. _librdkafka: https://github.com/edenhill/librdkafka
+
+Decision
+--------
+
+Instead of requiring confluent-kafka directly in base.in, we will wrap all imports of `confluent_kafka`in a `try...catch` block. If the import fails, the library will log an informative message and any calls will fail gracefully.
```

Comment:
Since this code could potentially be used in multiple modules, maybe we should add a more complete code example to copy of the import and the check.